src/openzaak/components/documenten/tests/test_cmis_url_mapping.py

```diff
@@ -37,6 +37,9 @@ def test_create_document_no_url_mapping(self):
         # Remove all available mappings
         UrlMapping.objects.all().delete()
 
+        Service.objects.create(
+            api_root="http://testserver/catalogi/api/v1/", api_type=APITypes.ztc
+        )
```

Comment:
I don't see this test failing if this is not added, why is it needed?

The only situation I can think of is that the informatieobjecttype is treated as a non-local URL, but instead for that you should set `ALLOWED_HOSTS=["testserver"]` (if that's not the case) as that's the mechanism used by loose-fk to determine if a resource is local or remote.